server/helpers/requests.js

```diff
@@ -0,0 +1,379 @@
+const { UnreachableCaseError } = require('./errors')
```

Comment:
We should probably note that this file is copied from `notifications` in case we want to figure out how to share code eventually.